src/hotspot/share/runtime/continuationEntry.hpp

```diff
@@ -53,10 +53,9 @@ class ContinuationEntry {
 
 public:
   static int return_pc_offset; // friend gen_continuation_enter
-  static void set_enter_nmethod(nmethod* nm); // friend SharedRuntime::generate_native_wrapper
+  static void set_enter_nmethod(CompiledMethod* nm); // friend SharedRuntime::generate_native_wrapper
```

Comment:
Do you need a forward declaration of CompiledMethod instead of nmethod at line 33?